docs/admin.md

```diff
@@ -0,0 +1,13 @@
+# Admin
+
+## Flags
+
+[`/admin/flags`](https://wpt.fyi/admin/flags) allows site-wide settings to be set, similar to the per-user [`/flags`](https://wpt.fyi/flags).
+
+## Flushing caches
+
+[`/admin/cache/flush`](https://wpt.fyi//admin/cache/flush) flushes some caches.
+
+## Uploading results
+
+[`/admin/results/upload`](https://wpt.fyi//admin/results/upload) is a form for upload results manually. Please file an issue on this project if you require credentials to upload results manually or via the API.
```

Comment:
@Hexcles what is the API endpoint? I tried to find a link to it but failed.

Reply:
It looks like one has to upload a gzipped report. Should that be documented inline, or here?

Reply:
The API is https://github.com/web-platform-tests/wpt.fyi/tree/master/api#results-creation

The file input on the upload UI has `accept=".gz"`.